.travis.yml

```diff
@@ -0,0 +1,22 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+dist: xenial
+
+language: java
+
+jdk:
```

Comment:
I'm not familiar with Travis+Github integration, is this is enough to trigger `mvn clean install` for commits pushed to a PR / master?
If so, then I'm +1 to merge this.

Reply:
AFAIK it should be, but I guess it wouldn't hurt to define this more explicitly.
Will merge with an updated version that defines the mvn command explicitly!

Reply:
Seems like the default command is `mvn test -B` only.
Will define jobs explicitly.